resources/views/community/live.blade.php

```diff
@@ -16,3 +16,73 @@
+                <div>
+                    <h1 class="livestream-header__headline">{{ trans('livestreams.top-headers.headline') }}</h1>
+                    <p class="livestream-header__description">{{ trans('livestreams.top-headers.description') }}</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="osu-layout__row osu-layout__row--with-gutter osu-layout__row--page livestream-page">
+    @if ($featuredStream != null)
+        <h2 class="livestream-page__header">{{ trans('livestreams.headers.featured') }}</h2>
+        <div class="livestream-featured">
+                <a class="livestream-page__anchor-twitch" href="{{$featuredStream->channel->url}}" target="_blank"> 
+                    <div class="wide col-sm-12 livestream-featured__container" style="background-image: url('{{$featuredStream->preview->large}}');">
+                        <div class="livestream-featured__info">
+                            <h3>{{$featuredStream->channel->name}}</h3>
+                            <p>{{$featuredStream->channel->status}}</p>
+                            <p>{{$featuredStream->viewers}} <i class="fa fa-eye"></i></p>
+                        </div>
+                    </div>
+                </a>
+                @if (Auth::user() != null && Auth::user()->isGmt())
+                <div class="forum-post__actions">
+                    <div class="forum-post-actions">
+                        <a data-method="POST" class="forum-post-actions__action" href="live?unpromote=true">
+                            <i class="fa fa-thumbs-down"></i>
+                        </a>
+                    </div>
+                </div>
+                @endif
+        </div>
+    @endif
+        <h2 class="livestream-page__header">{{ trans('livestreams.headers.regular') }}</h2>
+        @foreach ($streams as $stream)
+            <div class="wide col-sm-4 livestream-regular">
+                <a class="livestream-page__anchor-twitch" href="{{$stream->channel->url}}" target="_blank"> 
+                    <div class="livestream-regular__container">
+                        <div class="livestream-regular__top-background" style="background-image: url('{{$stream->preview->medium}}');">
+                            <div class="livestream-regular__streamer-info">
+                                {{$stream->channel->name}}
+                            </div>
+                            <div class="livestream-regular__watchers-info">
+                                <p>{{$stream->viewers}} <i class="fa fa-eye"></i></p>
+                            </div>
+                        </div>
+                        <div class="livestream-regular__bottom-info">
+                            {{$stream->channel->status}}
+                        </div>
+                    </div>
+                </a>
+                @if (Auth::user() != null && Auth::user()->isGmt())
+                <div class="forum-post__actions">
```

Comment:
the button positioning is a bit weird.

Also you're not supposed to use element from another block.